resources/views/commons/user_icon.blade.php

```diff
@@ -5,5 +5,4 @@
         $iconSrc = asset('storage/images/userIcons/' . $user->icon);
     }
```

Comment:
$userIcon = $this->userIcon($currentIcon);
if (!request()->file('icon')) {
                $userIcon = $currentIcon;
}